pkg/asset/machines/ibmcloud/zones.go

```diff
@@ -3,14 +3,15 @@ package ibmcloud
 import (
 	"context"
 
+	configv1 "github.com/openshift/api/config/v1"
 	"github.com/openshift/installer/pkg/asset/installconfig/ibmcloud"
```

Comment:
```suggestion

	"github.com/openshift/installer/pkg/asset/installconfig/ibmcloud"
```

Reply:
I believe these imports are formatted by `gofmt`, and making these changes results in that test failing
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_installer/7632/pull-ci-openshift-installer-master-gofmt/1721621569781895168

For example:
```
diff --git a/pkg/asset/machines/ibmcloud/zones.go b/pkg/asset/machines/ibmcloud/zones.go
index fe0fd4323f..d222b3a8dc 100644
--- a/pkg/asset/machines/ibmcloud/zones.go
+++ b/pkg/asset/machines/ibmcloud/zones.go
@@ -4,7 +4,6 @@ import (
 	"context"
 
 	configv1 "github.com/openshift/api/config/v1"
-
 	"github.com/openshift/installer/pkg/asset/installconfig/ibmcloud"
 )
```

Reply:
I'm going to resolve the `gofmt` items, but think that will result in the same imports as before attempting to address your comments in that regard.